src/components/BrandModelDropdowns.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect } from "react";
+import { UseFormRegister, UseFormWatch } from "react-hook-form";
+import { useModelsLazyQuery } from "../generated/graphql";
+import { IFormInputs } from "../pages/CarForm";
+import Dropdown from './Dropdown';
+
+interface DropdownProps {
+  brands: {
+    id: number,
+    name: string,
+  }[] | undefined;
+  isErrorBrand: boolean;
+  isErrorModel: boolean;
+  register: UseFormRegister<IFormInputs>;
+  watch: UseFormWatch<IFormInputs>
+}
+
+const BrandModelDropdowns = ({brands, isErrorBrand,isErrorModel, register, watch}: DropdownProps) => {
```

Comment:
I suggest you to read more about the hook form context, this way you can share all the methods from the form across multiple components without passing all the props that you could need: https://react-hook-form.com/advanced-usage/#ConnectForm